vivisect/__init__.py

```diff
@@ -955,16 +955,29 @@ def parseOpcode(self, va, arch=envi.ARCH_DEFAULT):
 
         return self.imem_archs[(arch & envi.ARCH_MASK) >> 16].archParseOpcode(b, off, va)
 
-    def iterJumpTable(self, startva):
+    def iterJumpTable(self, startva, step=None, maxiters=None):
+        if not step:
+            step = self.psize
+        iters = 0
         ptrbase = startva
         rdest = self.castPointer(ptrbase)
-        while self.isValidPointer(rdest):
+        STOP_LOCS = (LOC_STRING, LOC_UNI, LOC_STRUCT, LOC_CLSID, LOC_VFTABLE, LOC_IMPORT, LOC_PAD, LOC_NUMBER)
```

Comment:
this belongs outside the class-definition.  we don't want this to be instantiated each time it gets called.